Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php

```diff
@@ -32,6 +32,11 @@ public function testFixProvider()
         $template = '<?php %s';
         $tests = array();
         foreach (array('require', 'require_once', 'include', 'include_once') as $statement) {
+            $test[] = array(
+                sprintf('<?php %s "foo.php" ?>', $statement),
+                sprintf('<?php %s("foo.php") ?>', $statement),
```

Comment:
I assume this PR came out of this convesation: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1662#discussion_r49275979.